homeassistant/components/transmission/__init__.py

```diff
@@ -85,8 +85,7 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> b
     client = TransmissionClient(hass, config_entry)
     hass.data.setdefault(DOMAIN, {})[config_entry.entry_id] = client
 
-    if not await client.async_setup():
-        return False
+    await client.async_setup()
```

Comment:
I suggest that you amend the signature for `async_setup` in `TransmissionClient` to return `None` and remove the `return True` at the end of that method:
```python
async def async_setup(self) -> None:
```

Reply:
Thanks for the advise, code updated.